src/contracts/interfaces/IPaymentCoordinator.sol

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./IStrategy.sol";
+
+/**
+ * @title Interface for the `IPaymentCoordinator` contract.
+ * @author Layr Labs, Inc.
+ * @notice Terms of Service: https://docs.eigenlayer.xyz/overview/terms-of-service
+ */
+interface IPaymentCoordinator {
+
+    /// STRUCTS ///
+    struct StrategyAndMultiplier {
+        IStrategy strategy;
+        uint96 multiplier;
+    }
+	struct RangePayment {
+        // Strategies & relative weights of shares in the strategies
+        StrategyAndMultiplier strategiesAndMultlipliers;
+        IERC20 token;
+        uint256 amount;
+        uint256 startTimestamp;
+        // duration in seconds
+        uint256 duration;
+	}
+
+    struct DistributionRoot {
+        // timestamp after which the root can be claimed against
+        uint32 activatedAfter;
+        // merkle root of the distribution
+        bytes32 root;
+    }
+
+    struct ClaimsTreeMerkleLeaf {
+        IERC20 token; 
+        uint256 amount;
+        // Explicit recipient of the claim
+        address recipient;
```

Comment:
This we want the leaves to be specific to the owner, and we want the active claimer to be able to claim it. That is, the payments are attached to the owner and not the claimer.

Reply:
is this a comment about what to call the field?
I think `earner` would be most appropriate, given the other discussions I've had about this. it would be the staker/operator address, not necessarily the (only) address capable of making the claim.